tests/Unit/ServiceTest.php

```diff
@@ -216,6 +216,18 @@ public function testJsonSerialize()
                 'delivery_days'   => [
                     'Monday',
                 ],
+                'regions_from' => [
```

Comment:
Why did you add this here? Shouldn't region relationships have been removed all together?